features/dd-sdk-android-trace-otel/build.gradle.kts

```diff
@@ -61,6 +61,7 @@ dependencies {
     api(libs.openTelemetryApi)
     implementation(libs.kotlin)
     implementation(libs.androidXAnnotation)
+    implementation(libs.okHttp)
```

Comment:
this one shouldn't be here, there is no code change in `dd-sdk-android-trace-otel` module.

Reply:
yes was a left over from previous try outs :)